src/index.ts

```diff
@@ -16,23 +16,92 @@ app.use(bodyParser.urlencoded({ extended: false }));
 const db = new MockDB();
 const mariaDB = new MariaDB();
 
-app.get('/', (req, res) => res.send('Hello World!'))
-
-app.listen(port, () => console.log(`Example app listening on ${ip.address()}:${port}!`))
+app.get('/', (req, res) => res.send('Hello World!'));
 
+app.listen(port, () =>
+    console.log(`Example app listening on ${ip.address()}:${port}!`)
+);
 
 app.get('/stories', (req, res) => {
     res.send(JSON.stringify(db.getStories()));
-})
+});
 
 app.get('/review-reservations', (req, res) => {
     res.send(db.getReviewReservations());
-})
+});
+
+app.get('/reviews', (req, res) => {
+    mariaDB
+        .getReviews()
+        .then(reviews => {
+            res.send(reviews);
+        })
+        .catch(error => {
+            res.status(500).send(error.message);
+        });
+});
+
+app.post('/reviews', (req, res) => {
+    mariaDB
+        .addReview(req.body.reviewText, req.body.reviewerID, req.body.storyID)
+        .then(res.status(200).send())
+        .catch(error => {
+            console.log(error.message);
+            const msg = 'Review from this user already exists for this story';
+            if (error.message == msg) {
+                res.status(403).send(msg);
+            } else {
+                res.status(500).send('Internal Server Error');
+            }
+        });
+});
+
+app.post('/rating', (req, res) => {
+    mariaDB
+        .rateReview(req.body.reviewID, req.body.rating)
+        .then(res.status(200).send())
+        .catch(error => {
+            console.log(error.message);
+            const msg = 'Review does not exist';
+            if (error.message == msg) {
+                res.status(403).send(msg);
+            } else {
+                res.status(500).send('Internal Server Error');
+            }
+        });
+});
+
+app.get('/reservations', (req, res) => {
```

Comment:
modify so that you pass in a userID and it returns the reservations that that user owns